src/components/Calculator.js

```diff
@@ -0,0 +1,82 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <table className="container">
+      <tr className="row">
+        <td>
+          <input type="text" className="input-text" value="0" />
+        </td>
+      </tr>
+      <tr className="row">
+        <td>
+          <input type="button" value="AC" className="btn" />
+        </td>
+        <td>
+          <input type="button" value="+/-" className="btn" />
+        </td>
+        <td>
```

Comment:
- [ ] Please make sure to follow the design guidelines too, according to the requirements: 

![image](https://user-images.githubusercontent.com/77069011/218614678-e060ab7e-998c-4f7d-8db1-7d4575c3c113.png)

Your design:

![image](https://user-images.githubusercontent.com/77069011/218614728-39646522-36e3-4cb1-92bb-249b70d8f377.png)

Remember that following a given design is one of the main skills that you're going to apply on a daily basis in the industry, and having a good-looking and consistent design can attract the attention of future recruiters! 🤓 🔧

Reply:
done